src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt

```diff
@@ -76,11 +108,12 @@ object SectionSyncListener : Listener {
         blockEvent.newState.location.sync()
     }*/
 
-    //TODO this causes duplication glitches that need to be fixed first
-    /*@EventHandler
-    fun onBlockMultiPlaceEvent(blockEvent: BlockMultiPlaceEvent) {
-        blockEvent.replacedBlockStates.copyBlocks()
-    }*/
+    @EventHandler
+    fun BlockMultiPlaceEvent.syncMultiBlockPlace() {
+        for (blockState in replacedBlockStates) {
```

Comment:
Should we sync blocks regardless of what it's state is or only sync the ones we know we have handled state for? (beds, and bisected?)

Reply:
They are the only multiblock blocks, aren't they?